src/core/base.js

```diff
@@ -1,18 +1,20 @@
 'use strict';
-const _ = require('lodash');
+
 const sjcl = require('./utils').sjcl;
-const utils = require('./utils');
+const sha256 = require('./utils').sha256;
 const extend = require('extend');
-const convertBase = require('./baseconverter');
 
 const Base = {};
 
 const alphabets = Base.alphabets = {
+  bitcoin: '123456789ABCDEFGHJKLMNPQRSTUVWXYZabcdefghijkmnopqrstuvwxyz',
   ripple: 'rpshnaf39wBUDNEGHJKLM4PQRST7VWXYZ2bcdeCg65jkm8oFqi1tuvAxyz',
-  tipple: 'RPShNAF39wBUDnEGHJKLM4pQrsT7VWXYZ2bcdeCg65jkm8ofqi1tuvaxyz',
-  bitcoin: '123456789ABCDEFGHJKLMNPQRSTUVWXYZabcdefghijkmnopqrstuvwxyz'
+  tipple: 'RPShNAF39wBUDnEGHJKLM4pQrsT7VWXYZ2bcdeCg65jkm8ofqi1tuvaxyz'
 };
 
+const codecOptions = {sha256, alphabets, defaultAlphabet: 'ripple'};
```

Comment:
I don't think there's much benefit to having symbolic names for alphabets considering that each application is probably only going to use one of them and `ripple-address-codec` will already have it specified. Having symbolic names means that you have to add an additional method to the interface to register alphabets rather than being able to pass them in directly.